pkg/restapi/device/operations.go

```diff
@@ -296,11 +297,15 @@ func (o *Operation) getUserData(w http.ResponseWriter, r *http.Request) (userDat
 	userData, err = o.store.users.Get(fmt.Sprintf("%v", userSub))
 	if err != nil {
 		common.WriteErrorResponsef(w, logger,
-			http.StatusInternalServerError, "failed to read user session cookie: %s", err.Error())
+			http.StatusInternalServerError, "failed to get user data %s:", err.Error())
 
 		return nil, false
 	}
 
+	displayName := strings.Split(fmt.Sprintf("%v", userSub), "@")[0]
+
+	userData.FamilyName = displayName
```

Comment:
FamilyName == displayName ?

Reply:
we had to pick one from our existing user data stuct (FamilyName, Name, Sub, Email and GivenName ) equivalent to device display name. 
func NewDevice(usr *user.User) *Device {
	device := &Device{}
	device.ID = usr.Sub
	device.Name = usr.Name
	device.DisplayName = usr.FamilyName
	device.Credentials = []webauthn.Credential{}

	return device
}

Yes right now FamilyName == displayName